### PR TITLE
Adopt UT test 09605

### PR DESCRIPTION
**Pull request by guy-maurel**
Thanks to Antti Tapaninen

### CODE REVIEW

File: tests/config/aet-2.cfg [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+pp_indent                                            = ignore
+pp_indent_at_level                                   = false
+pp_indent_count                                      = 1
+pp_space                                             = ignore
+pp_space_count                                       = 0
+pp_indent_region                                     = 0
+pp_region_indent_code                                = false
+pp_indent_if                                         = 0
+pp_if_indent_code                                    = false
+pp_define_at_level                                   = false
```
Comment (CDanU) [inline tests/config/aet-2.cfg]: needs to be reduced

File: tests/input/c/negative_value.c [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-	int frames = 127;
-	frames = ((frames + 15) & -16);
-	return 0;
-}
\ No newline at end of file
+    int frames = 127;
+    frames = ((frames + 15) & - 16);
+    return 0;
+}
```
Comment (CDanU) [inline tests/input/c/negative_value.c]: This change breaks the test as it adds a space after the '-'. 
The test is supposed to test the behavior of a minus that is attached to a digit (CT_NEG vs. CT_ARITH)

File: tests/c.test [hunk 3]
```diff
@@ -352,7 +352,9 @@
 09604  indent_ternary_operator-2.cfg        c/indent_ternary-2.c
 
 09605  sp_arith-f.cfg                       c/negative_value.c
+09606  aet-2.cfg                            c/extern-c-attribute.c
 
+09609  aet-1.cfg                            c/negative_value.c
```
Comment (CDanU) [inline tests/c.test]: This is handled by test 09605

File: tests/config/aet-1.cfg [hunk 4]
```diff
@@ -0,0 +1 @@
+indent_columns                  = 4
```
Comment (CDanU) [inline tests/config/aet-1.cfg]: -> this file can therefore be removed

File: tests/output/c/09605-negative_value.c [hunk 5]
```diff
@@ -7,4 +7,4 @@ int main(void) {
 	int frames = 127;
 	frames = ((frames + 15) & -16);
 	return 0;
-}
\ No newline at end of file
+}
```
Comment (CDanU) [inline tests/output/c/09605-negative_value.c]: this newline shouldn't be there

File: src/combine.cpp [hunk 6]
```diff
@@ -1860,6 +1860,41 @@ void fix_symbols(void)
...
+
+      if ((pc->type == CT_ATTRIBUTE) && (cpd.lang_flags & (LANG_ALLC)))
+      {
+         chunk_t *tmp = skip_attribute_next(pc);
+         if ((tmp != NULL) && (tmp->type == CT_WORD))
+         {
+            tmp->flags |= PCF_STMT_START | PCF_EXPR_START;
+         }
+      }
+
```
Comment (CDanU) [inline src/combine.cpp]: I had cleaned up this code and the tests for it already, but for some reason I haven't made a PR for that
give me some time to check it again

**Comment by guy-maurel** [comment]
Well, I'll wait for you.

**Comment by CDanU** [comment]
This one can be closed now as 1586 was merged

**Pull request closed without merging**